internal/auth/middleware.go

```
@@ -7,7 +7,10 @@ import (
 
 type contextKey string
 
-const userContextKey contextKey = "user"
+const (
+	userContextKey  contextKey = "user"
```

golangci-lint (2.12.2)

[error] SA9004: only the first constant in this group has an explicit type

(staticcheck)